Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Sliders/PinchSlider.cs

```diff
@@ -33,20 +35,66 @@ public GameObject ThumbRoot
             }
         }
 
-        [Range(0, 1)]
+        #region MyValue 
+        [SerializeField]
+        private int minValue = -4;
+        [SerializeField]
+        private int maxValue = 4;
+
+        [SerializeField]
+        private bool wholeNumbers = true;
```

Comment:
It would be awesome to have a comment for this (i.e. explaining the significance of wholeNumbers) - i.e. if this is false, I assume that myValue can end up being a non-whole number between minValue and maxValue.

If this is true, is it valid for startValue to be a non-wholeNumber?

There are some interesting interactions that usually come about when there are bools like this, and it's important to get those documented